processor/probabilisticsamplerprocessor/go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	go.opentelemetry.io/otel/metric v1.27.0
 	go.opentelemetry.io/otel/trace v1.27.0
 	go.uber.org/goleak v1.3.0
+	go.uber.org/multierr v1.11.0
```

Comment:
I think we are avoiding this one now, using stdlib instead.

Reply:
Thanks, now using `errors.Join`. 95ecbaede7